### DISCUSSION TITLE
Is it possible to manage how much space there is between the code and the comment on that same line?

### DISCUSSION THREAD

**Discussion by ghost** [prettier/prettier#15170] [Q&A]
What i wrote:
`let variable = 3;    // Comment`

Expected result when saving the file:
`let variable = 3;    // Comment`

Actual result when saving the file:
`let variable = 3; // Comment`

I was wondering if it is possible to decide how much space we want to set as i'm looking to give more space between the actual working code and a comment on the same line.

**Comment by kachkaev** [comment]
👋 @M4rkid!

> if it is possible to decide how much space we want to set

See https://prettier.io/docs/en/why-prettier.html

> By far the biggest reason for adopting Prettier is to stop all the on-going debates over styles.
